e2e/server/app.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+import _ from 'lodash'
+import E2EExample from './E2EExample'
+import routes from './routes'
+import { BrowserRouter, Route, Switch, Link } from 'react-router-dom'
+import { Provider, themes, Flex, Header, List } from '@stardust-ui/react'
+
+const ContentList = () => (
+  <>
+    <Flex column hAlign="center">
+      <Header as="h1" content="E2E Test Examples" />
+      <List
+        items={_.keys(routes).map(testExampleUrl => ({
+          key: testExampleUrl,
+          header: <Link to={testExampleUrl}>{_.startCase(testExampleUrl)}</Link>,
+        }))}
+      />
+    </Flex>
+  </>
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/59342116-69c12b00-8d09-11e9-9f4a-18f55facb027.png)

Suggest to change to:

```
  <Flex column hAlign="center">
    <Header as="h1" content="E2E Test Examples" />
    <List
      items={_.keys(routes).map(testExampleUrl => ({
        key: testExampleUrl,
        header: { as: Link, to: testExampleUrl, content: _.startCase(testExampleUrl) },
      }))}
    />
  </Flex>
```

Reply:
another proof of our unsafe API for shorthand as React element..